CMakeLists.txt

```diff
@@ -25,11 +25,20 @@ target_compile_definitions( DemoGafferExtensionModule PRIVATE BOOST_SIGNALS_NO_D
 target_link_libraries( DemoGafferExtensionModule DemoGafferExtension )
 
 if(APPLE)
-  target_include_directories( DemoGafferExtensionModule PRIVATE include ${GAFFER_ROOT}/include ${GAFFER_ROOT}/lib/Python.framework/Versions/2.7/include/python2.7 )
+  if( "${PYTHON_VERSION}" MATCHES "3.7" )
+    target_include_directories( DemoGafferExtensionModule PRIVATE include ${GAFFER_ROOT}/include ${GAFFER_ROOT}/lib/Python.framework/Versions/3.7/include/python3.7 )
+  else()
+    target_include_directories( DemoGafferExtensionModule PRIVATE include ${GAFFER_ROOT}/include ${GAFFER_ROOT}/lib/Python.framework/Versions/2.7/include/python2.7 )
+  endif()
```

Comment:
This should be possible without an `if` by just using `${PYTHON_VERSION}` in the path directly, right?